etc/notebooks/WidgetInitTest.ipynb

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Did you mean to checkin this file here? should it be under `etc/notebooks/tests/WidgetInitTest.ipynb`